samples/nrf9160/location/overlay-pgps.conf

```diff
@@ -11,6 +11,9 @@ CONFIG_NRF_CLOUD_REST=y
 CONFIG_NRF_CLOUD_PGPS=y
 CONFIG_NRF_CLOUD_PGPS_REPLACEMENT_THRESHOLD=4
 CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT=n
```

Comment:
I started thinking that we don't probably need to set CONFIG_NRF_CLOUD_PGPS_REQUEST_UPON_INIT=n. I think it's a leftover from the time when location library initialized P-GPS when location lib was initialized at application startup and currently that is done only at the first attempt to request the location. I tested that it works without this line, so unless you can think of any reason to keep it would you mind removing it in this PR? Thanks!

Reply:
It works either way, so I'll remove it.

I did notice that the first GNSS fix's timeout period starts too soon, so by the time the A-GPS and P-GPS data are both injected, it times out less than a second later.  Maybe it would be better for the location library to wait until assistance is actually loaded into the modem before starting the timer?

Reply:
> I did notice that the first GNSS fix's timeout period starts too soon, so by the time the A-GPS and P-GPS data are both injected, it times out less than a second later. Maybe it would be better for the location library to wait until assistance is actually loaded into the modem before starting the timer?

If you talk about the Location sample, the first GNSS attempt is intended to time out to force fallback to cellular. It also says that in the sample output: "Requesting location with short GNSS timeout to trigger fallback to cellular".

Reply:
Yes, the location sample.  OK, that makes sense.  Thanks.